src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -237,7 +237,7 @@
 examples:
   - name: Add client certificate by thumbprint
     text: >
-        az sf cluster client-certificate add -g group-name -c cluster1 --thumbprint '5F3660C715EBBDA31DB1FFDCF508302348DE8E7A'
+        "az sf cluster client-certificate add -g testRG -c testCluster --certificate-common-name testCommonName --certificate-issuer-thumbprint AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
```

Comment:
Why do you add double quotation marks to the sample?

Reply:
updated this